scrapers/czechcasting.yml

```
@@ -0,0 +1,23 @@
+name: czechav
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - czechcasting.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $studio: //*[@id="logo"]/a/img/@alt
+    scene:
+      Title: /html/head/title/text()
+      Details: //*[@id="girl-info"]/div/div/div[2]/p/text()
+      Tags:
+        Name: //*[@id="girl-info"]/div/div/ul[@class='tags']/li/a[@class='tag']/text()
+      Studio:
+        Name: $studio
+      Image:
+        selector: //video/@poster
+      Date:
+        selector: //p[@class='updated']/text()
```

the date need to be in the format that stash expects so we need some extra postprocess
```suggestion
      Date:
        selector: //p[@class='updated']/text()
        postProcess:
          - parseDate: 01/02/2006
```